docs_src/design/ucr/URIs-for-Files.md

```diff
@@ -0,0 +1,56 @@
+## URIs for files
+### Submitters
+- Lenny Goodell (Intel)
+
+## Change Log
+- [pending](URL of PR) (2023-01-18)
+
+### Market Segments
+- All
+
+### Motivation
+Deployment at scale, i.e. identical or almost identical deployments across many locations, would benefit from the ability to loaded service files from a central location. This would allow the maintainer to make changes once to a shared file and have them apply to all or a subset of deployments. The following are some EdgeX service files that would benefit for this capability:
```

Comment:
```suggestion
Deployment at scale, i.e. identical or almost identical deployments across many locations, would benefit from the ability to load service files from a central location. This would allow the maintainer to make changes once to a shared file and have them apply to all or a subset of deployments. The following are some EdgeX service files that would benefit for this capability:
```

Reply:
Another config:

<https://github.com/edgexfoundry/edgex-go/blob/main/cmd/security-file-token-provider/res/token-config.json>

specifies the pre-created service list and custom input parameters to their Vault policies.

Reply:
Love it! Added